app/uk/gov/hmrc/tariffclassificationfrontend/views/components/initial_focus.scala.html

```diff
@@ -14,15 +14,18 @@
  * limitations under the License.
  *@
 
-@(focusOnThisElement: Option[String] = None)
+@(focusOnThisElement: Option[String] = None, withTimeout: Int = 0)
 
 <script>
-    var hasErrors = document.getElementById("error-summary");
-    if (hasErrors) {
-        hasErrors.focus();
-    } else {
-        @if(focusOnThisElement.isDefined){
-             document.getElementById("@focusOnThisElement").focus();
-        }
-    }
+        setTimeout(function() {
+            var hasErrors = document.getElementById("error-summary");
+            if (hasErrors) {
+                hasErrors.focus();
+            } else {
+                @if(focusOnThisElement.isDefined){
+                     document.getElementById("@focusOnThisElement").focus();
+                }
+            }
+                }
+    , @withTimeout);
```

Comment:
This timeout has been introduced because we want to select the component of choice with a slight delay on the search pages because there's some other javascript present that grabs the focus for the keywords box, and we want to do it after that to ensure we win.